lib/tasks/dev.rake

```diff
@@ -31,6 +33,70 @@ namespace :dev do
     Kernel.puts "===="
   end
 
+  # protip: set EMAIL_ENCRYPTION_COST and SCRYPT_COST env vars to '800$8$1$'
+  # or whatever the test env uses, to override config/application.yml
+  # before running this task.
+  # e.g.
+  # rake dev:random_users NUM_USERS=1000 SCRYPT_COST='800$8$1$' EMAIL_ENCRYPTION_COST='800$8$1$'
+
+  # some baseline metrics
+  # $ rake dev:random_users NUM_USERS=1000 SCRYPT_COST='800$8$1$' EMAIL_ENCRYPTION_COST='800$8$1$'
+  # Users: 100% |==================================================| Time: 00:00:37
+  # $ rake dev:random_users NUM_USERS=100000 SCRYPT_COST='800$8$1$' EMAIL_ENCRYPTION_COST='800$8$1$'
+  # Users: 100% |==================================================| Time: 01:06:08
+
+  desc 'Create N random User records'
+  task random_users: :environment do
+    pw = 'salty pickles'
+    num_users = (ENV['NUM_USERS'] || 100).to_i
+    num_created = 0
+    progress = ::ANSI::Progressbar.new("Users", num_users, STDOUT)
+    win_width = progress.__send__ :get_width
+    title_width = (win_width / 4).to_i
+    progress.format("Users: %3d%% %s %s", :percentage, :bar, :stat)
+    progress.__send__ :show
+    progress.bar_mark = '='
+
+    User.transaction do
+
+      while (num_created < num_users) do
+        email_addr = "testuser#{num_created}@example.com"
+        ee = EncryptedEmail.new_from_email(email_addr)
+        User.find_or_create_by!(email_fingerprint: ee.fingerprint) do |user|
+          user.encrypted_email = ee.encrypted
+          user.skip_confirmation!
+          user.reset_password(pw, pw)
+          user.phone = format('+1 (415) 555-%04d', num_created)
+          user.phone_confirmed_at = Time.current
+          Event.create(user_id: user.id, event_type: :account_created)
+        end
+
+        if ENV['VERIFIED']
+          user = User.find_by(email_fingerprint: ee.fingerprint)
+          profile = Profile.new(user: user)
+          pii = Pii::Attributes.new_from_hash(
+            first_name: 'Test',
+            last_name: "User #{num_created}",
+            dob: '1970-05-01',
+            ssn: "666-#{num_created}" # doesn't need to be legit 9 digits, just unique
+          )
+          recovery_code = profile.encrypt_pii(loa3_user.user_access_key, pii)
+          profile.activate
+
+          Kernel.puts "===="
```

Comment:
I know we already do `Kernel.puts` but I think we may want to switch to `Rails.logger.info` or something? One of the tests runs the task above and it's annoying to see the output there